.github/workflows/scripts/pending-prs

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+export BROWSER=echo
+current_directory=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
+format="$1"
+repo="$2"
+branches_file="$current_directory/../../branches.yml"
+
+readarray activeBranches < <(yq e -o=j -I=0 '.active[]' "$branches_file")
+
+get_url() {
+    local repo=$1
+    local branch=$2
+    gh search prs --repo=$repo --state=open --base $branch -w | awk '{print substr($0, 1, length($0)-12)}' 2> /dev/null
```

Comment:
Btw, the nasty hack with `awk` here is b/c by default the `gh` client likes to mis-append an `&type=issues` onto the end of the URL which breaks it when trying to jump to the in-browser url:

```
➜  redpanda-operator git:(as/more-actions) BROWSER=echo gh search prs --repo=redpanda-data/redpanda-operator --state=open --base main -w 2> /dev/null
https://github.com/search?q=base%3Amain+repo%3Aredpanda-data%2Fredpanda-operator+state%3Aopen+type%3Apr&type=issues
```

See the difference in the [broken output](https://github.com/search?q=base%3Amain+repo%3Aredpanda-data%2Fredpanda-operator+state%3Aopen+type%3Apr&type=issues) v. [fixed](https://github.com/search?q=base%3Amain+repo%3Aredpanda-data%2Fredpanda-operator+state%3Aopen+type%3Apr)